PackageConfig.cmake

```diff
@@ -384,9 +387,14 @@ configure_file(
 )
 
 # create and install ProjectConfigVersion.cmake
-write_basic_package_version_file(
-  ${PROJECT_BINARY_DIR}/pkg/${PROJECT_NAME}ConfigVersion.cmake
-  VERSION ${VERSION_MAJOR}.${VERSION_MINOR} COMPATIBILITY SameMajorVersion)
+if(${UPPER_PROJECT_NAME}_CONFIG_VERSION_IN)
+  set(PACKAGE_VERSION "${VERSION_MAJOR}.${VERSION_MINOR}")
```

Comment:
PACKAGE_VERSION is set here so that it is available to the template file; analogous to the setting of CVF_PACKAGE_VERSION for BasicConfigVersion-XXXX.cmake.in.